test/backend/routes.spec.js

```diff
@@ -336,20 +367,20 @@ describe('Authentication - ', () => {
         .then(res => {
             // request should be allowed:
             GET('settings').then(res => {
-              console.log('status1:' + res.status);
-              assert.equal(res.status, 200);
+                assert.equal(res.status, 200);
+                return;
```

Comment:
I've created a little promise example for you to play around with so that it is crystal clear how this works: https://codepen.io/chriddyp/pen/boMNEz?editors=0011